lang/cs/Org.Apache.REEF.Tang/Formats/ConfigurationFile.cs

```diff
@@ -100,7 +100,7 @@ public class ConfigurationFile
         //    {
         //        serializedString = reader.ReadLine();
         //    }
-        //    AvroDeseriaize(conf, serializedString);
+        //    AvroDeserialize(conf, serializedString);
         //}
         //#endregion Avro serialization
```

Comment:
This whole commented out block is unused, we're using AvroConfigurationSerializer instead. I think it's easier just to delete these comments than to fix typos in them